src/main/java/roomescape/global/config/TossRestClientConfig.java

```diff
@@ -0,0 +1,40 @@
+package roomescape.global.config;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestClient;
+
+
+@Configuration
+public class TossRestClientConfig {
+
+    private final String secretKey;
+    private final String baseUrl;
+
+    public TossRestClientConfig(@Value("${toss.payment.secret-key}") String secretKey, @Value("${toss.payment.base-url}") String baseUrl) {
+        this.secretKey = secretKey;
+        this.baseUrl = baseUrl;
+    }
+
+    @Bean
+    public RestClient restClient() {
+        String basicAuthValue = encodeBasicAuth(secretKey);
+
+        return RestClient.builder()
+                .baseUrl(baseUrl)
+                .defaultHeader("Authorization", basicAuthValue)
+                .defaultHeader("Content-Type", APPLICATION_JSON_VALUE)
```

Comment:
```suggestion
                .defaultHeader(HttpHeaders.AUTHORIZATION, basicAuthValue)
                .defaultHeader(HttpHeaders.CONTENT_TYPE, APPLICATION_JSON_VALUE)
```
이미 정의된 상수를 활용해 볼 수 있을 것 같아요.

Reply:
이미 정의된 상수가 있었군요! 감사합니다!